### PR TITLE
kubesess: add shell functions and completions.

### PR DESCRIPTION
**Pull request by Ramilito**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/kubesess.rb [hunk 1]
```diff
@@ -19,6 +19,25 @@ class Kubesess < Formula
...
+    fish_function.install "./scripts/fish/functions/kcd.fish"
+    fish_function.install "./scripts/fish/functions/kn.fish"
+    fish_function.install "./scripts/fish/functions/knd.fish"
+  end
+
+  def caveats
+    <<~EOS
+      To use kubesess it is required to enable completions.
+         Read more here: https://docs.brew.sh/Shell-Completion
+    EOS
```
Comment (carlocab) [inline Formula/kubesess.rb:40]:
```suggestion
```
This isn't specific to Homebrew.

File: Formula/kubesess.rb [hunk 2]
```diff
@@ -19,6 +19,25 @@ class Kubesess < Formula
...
+
+    fish_completion.install "./scripts/fish/completions/kc.fish"
+    fish_completion.install "./scripts/fish/completions/kn.fish"
+    fish_completion.install "./scripts/fish/completions/knd.fish"
+    fish_completion.install "./scripts/fish/completions/kcd.fish"
+
+    fish_function.install "./scripts/fish/functions/kc.fish"
+    fish_function.install "./scripts/fish/functions/kcd.fish"
+    fish_function.install "./scripts/fish/functions/kn.fish"
+    fish_function.install "./scripts/fish/functions/knd.fish"
```
Comment (carlocab) [inline Formula/kubesess.rb:33]:
```suggestion
    bash_completion.install "scripts/sh/completion.sh"
    zsh_function.install "scripts/sh/kubesess.sh"

    %w[kc kn knd kcd].each do |basename|
      fish_completion.install "scripts/fish/completions/#{basename}.fish"
      fish_function.install "scripts/fish/functions/#{basename}.fish"
    end
```

**Review by carlocab — CHANGES_REQUESTED** [review]
Please use the preferred commit-message style for homebrew/core. We put the name of the formula first in commit-message headings.

For new formulae:

> At Homebrew, we like to put the name of the formula up front like so: `foobar 7.3 (new formula)`.

For existing formulae:

> The preferred commit message format for simple version updates is `foobar 7.3` and for fixes is `foobar: fix flibble matrix.`.

Refer to the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit) for more details. Also, when making further changes to your pull request, use the following guidelines to make sure that @BrewTestBot can merge your commits:

> - One formula per commit; one commit per formula.
> - Keep merge commits out of the pull request.

Also:

Please bump the `revision` (either add `revision 1` or increment the `revision` right after `license`) so that users who have this installed get the updated version of the formula when they do `brew upgrade`. Thanks!

**Comment by carlocab** [comment]
You can use a commit message like:
```
kubesess: add shell functions and completions.
```

File: Formula/kubesess.rb (hunk 1 above)
Comment (Ramilito) [reply Formula/kubesess.rb]: Removed this.

File: Formula/kubesess.rb (hunk 2 above)
Comment (Ramilito) [reply Formula/kubesess.rb]: Added revison 1

Replaced my ugly code with your pretty one.

Comment (Ramilito) [reply Formula/kubesess.rb]: Also fixed the commit message to suit the commit style guide, don't know how I forgot about that.

File: Formula/kubesess.rb (hunk 1 above)
Comment (Ramilito) [reply Formula/kubesess.rb]: I will add the specifics in the installation instructions instead.

**Review by Ramilito — COMMENTED** [review]
Fixed!

**Review by Ramilito — COMMENTED** [review]
Fixed!

**Comment by Ramilito** [comment]
I'm not sure how to remove the change requested status, the suggested changes have been applied though!

**Review by carlocab — APPROVED** [review]
Thanks!

**Pull request closed without merging**